src/pudl/io_managers.py

```diff
@@ -309,52 +310,39 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
 class PudlParquetIOManager(IOManager):
     """IOManager that writes pudl tables to pyarrow parquet files."""
 
-    def handle_output(
-        self, context: OutputContext, df: pd.DataFrame | gpd.GeoDataFrame
-    ) -> None:
+    def handle_output(self, context: OutputContext, obj: pd.DataFrame) -> None:
         """Writes pudl dataframe to parquet file."""
-        if not isinstance(df, (pd.DataFrame, gpd.GeoDataFrame)):
-            raise TypeError(
-                f"Only pandas or geopandas dataframes are supported, got {type(df)}."
-            )
+        if not isinstance(obj, pd.DataFrame):
+            raise TypeError(f"Only pandas dataframes are supported, got {type(obj)}.")
         table_name = get_table_name_from_context(context)
         parquet_path = PudlPaths().parquet_path(table_name)
         parquet_path.parent.mkdir(parents=True, exist_ok=True)
 
         res = Resource.from_id(table_name)
-        df = res.enforce_schema(df)
+        df = res.enforce_schema(obj)
         pa_schema = res.to_pyarrow()
+        # Test whether these are actually necessary, or if we can use df.to_parquet()
```

Comment:
Looks like we *can* use `df.to_parquet()`? Or is that waiting on branch builds for verification?

Reply:
Whoops yes! Seems to work fine. Will remove.